### PR TITLE
🌐 Add Japanese translation for `docs/ja/docs/tutorial/dependencies/dependencies-with-yield.md`

### PR DESCRIPTION
**Pull request by SwftAlpc**
This PR translates the tutorial/dependencies/dependencies-with-yield.md to Japanese.
Please review tutorial/dependencies/dependencies-with-yield.md.

~Issue #1571 generates a new directory to support Japanese.
PR #1656 generates a new directory for tutorial.~
PR #1957 generates a new directory for dependencies in tutorial.
I will fix dependency after ~#1571, #1656 and~ #1957 are merged.

### CODE REVIEW

**Comment by SwftAlpc** [comment]
@tokusumi 
Thanks for your review!
I adopt your suggetions.
Please check it out!

**Review by tokusumi — APPROVED** [review]
Thank you for cooperating for improvement.

File: docs/ja/docs/tutorial/dependencies/dependencies-with-yield.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+!!! note "技術詳細"
+    これはPythonの<a href="https://docs.python.org/3/library/contextlib.html" class="external-link" target="_blank">Context Managers</a>のおかげで動作します。
+
+    **FastAPI** はこれを実現するために内部的に使用しています。
+
+## `yield`と`HTTPException`を持つ依存関係
+
+`yield`と例外をキャッチする`try`ブロックを持つことができる依存関係を使用することができることがわかりました。
+
+`yield`の後の終了コードで`HTTPException`などを発生させたくなるかもしれません。しかし**それはうまくいきません**
```
Comment (sh0nk) [inline docs/ja/docs/tutorial/dependencies/dependencies-with-yield.md:109]:
```suggestion
`yield`の後の終了コードで`HTTPException`などを発生させたくなるかもしれません。しかし**それはうまくいきません**。
```
Add a trailing period.

File: docs/ja/docs/tutorial/dependencies/dependencies-with-yield.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+`yield`を持つ依存関係の終了コードは[例外ハンドラ](../handling-errors.md#install-custom-exception-handlers){.internal-link target=_blank}の*後に*実行されます。依存関係によって投げられた例外を終了コード（`yield`の後）でキャッチするものはなにもありません。
+
+つまり、`yield`の後に`HTTPException`を発生させた場合、`HTTTPException`をキャッチしてHTTP 400のレスポンスを返すデフォルトの（あるいは任意のカスタムの）例外ハンドラは、その例外をキャッチすることができなくなります。
```
Comment (sh0nk) [inline docs/ja/docs/tutorial/dependencies/dependencies-with-yield.md:113]:
```suggestion
つまり、`yield`の後に`HTTPException`を発生させた場合、`HTTPException `をキャッチしてHTTP 400のレスポンスを返すデフォルトの（あるいは任意のカスタムの）例外ハンドラは、その例外をキャッチすることができなくなります。
```
typo

**Review by sh0nk — COMMENTED** [review]
Could you fix only the typos?

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1cfe4ba6ff1af5cfd87bafaae244026233fd7949 at: https://6499cbcd89d3ab061a01d4fc--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this was done by @SwftAlpc and approved by @tokusumi, champions of the Japanese translations, I'll merge this right away. 🚀 Thanks! 🍪

**Pull request merged**